index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+  <title>Easy Science</title>
+  <script src="script.js" defer></script>
+</head>
+<body>
+  <header>
+    <section class="social-network">
+      <ul>
+        <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+        <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+        <li><a href="#">Español</a></li>
+      </ul>
+    </section>
+    <nav>
+      <div class="logo"><a href="index.html"><strong>E</strong>asy <strong>S</strong>ience</a></div>
+      <div class="hamburger-menu"><img id="menu-button" src="assets/hamburger-menu.png" alt="menu"></div>
+      <ul id="menu">
+        <li class="closeButtonContainer"><span id="closeButton">X</span></li>
+        <li><a class="Personal-information" href="courses.html">Courses</a></li>
+        <li><a class="Personal-information" href="#about">About</a></li>
+        <li><a class="Personal-information" href="#contact">Contact</a></li>
+        <li><a class="Personal-information button sign-up" href="#">Sign up</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="home-page">
+        <div class="home-text">
+          <h1>We teach science the right way</h1>
+          <p>We have the mission to provide a world-class education for anyone, anywhere.</p>
+          <a class="button" href="courses.html">Start here</a>
+        </div>
+    </section>
+    <section id="about" class="about">
+      <h2>Find a comprehensive explanation for the subject you've been struggling with</h2>
+      <hr>
+      <div>
+        <p>Start with:</p>
+        <ul>
+          <li class="button"><a href="#">Math   <i class="fas fa-square-root-alt"></i></a></li>
+          <li class="button"><a href="#">Physics   <i class="fas fa-atom"></i></a></li>
+          <li class="button"><a href="#">Chemistry   <i class="fas fa-flask"></i></a></li>
+        </ul>
+      </div>
+    </section>
+    <section id="courses" class="courses-container">
+      <h1>Take a look at our most popular courses</h1>
+      <hr>
+      <section id="courses-wrap" class="courses">
+        <div class="button-container">
+          <a class="button show-more" href="#">Show more   <i class="fas fa-arrow-down"></i></a>
+          <a class="button" href="courses.html">Find more courses   <i class="fas fa-book-reader"></i></a>
+        </div>
+      </section>
+    </section>
```

Comment:
- [ ] [REQUIRED] I think the next section should include a column that has a section header, a divider, and six individual articles, each with an image, title, subtitle, and description. Each article has two columns, the first with the image at the left and the second with the title, subtitle, and description. 
- [ ] This content is generated dynamically with JavaScript.
- [ ] Let's also not forget to follow the design below

![image](https://user-images.githubusercontent.com/4247740/147305073-bf6f0288-d051-45a6-80ae-25f81285a086.png)

Reply:
## Thanks for the feedback ##

<img width="1188" alt="Captura de Pantalla 2021-12-24 a la(s) 12 46 29" src="https://user-images.githubusercontent.com/36294012/147369227-deadae47-f85c-4f88-80bd-f568ecbaff90.png">

This is how it looks now when you hover!
<img src="https://media3.giphy.com/media/TPqd1b3bE0oTbLbMmb/giphy.gif"/>